### PR TITLE
Tenants with application not registered proceeded right now

### PR DESCRIPTION
**Pull request by bergano65**

### CODE REVIEW

File: Shared/AzureResourceManagerUtil.cs [hunk 1]
```diff
@@ -67,15 +67,39 @@ public static List<Organization> GetUserOrganizations()
                     string responseContent = response.Content.ReadAsStringAsync().Result;
                     var organizationsResult = (Json.Decode(responseContent)).value;
 
+                    string objectIdOfCloudSenseServicePrincipal = null;
+
                     foreach (var organization in organizationsResult)
```
Comment (shklar) [inline Shared/AzureResourceManagerUtil.cs]: We already know the ID of AAD.
Why not just add it to our configuration?

**Comment by bergano65** [comment]
If somebody manually will authorize our application at his tenant this code will work automatically. Still think we should hard code it?

**Comment by shklar** [comment]
Yes.
Tho is not a scenario we are targeting right now.

Sent from my Windows Phone
________________________________

**Comment by bergano65** [comment]
ok

**Comment by bergano65** [comment]
Pls inspect changes by your scenario

**Pull request merged**